### DISCUSSION TITLE
v29 14.08.2023

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2nr_unofficial_update#526] [Announcements]
- Исправлена ситуация, когда Гаан не давал обещанную награду за убийство фыркающего снеппера.
- Исправлен перевод рецепта Молота Лу (спасибо Єгорд).
- Добавлен завершающий диалог с Буллко после сопровождения ГГ к стоянке у реки.
- Если противник погиб от падения после заклинания 'Кулак ветра', то ГГ будет начислен опыт.
- Сифер правильно атакует Дара, если рассказать о нем ночью.
- Мелдор и Мо будут атаковать ГГ в случае убийства Лемара.
- Герольд будет съедать припасы, которые приносит ему ГГ.
- Квест 'Доспехи бандитов' будет выполнен при получении доспехов от Бонеса.
- Добавлена возможность отдать учетную книгу Фрэнсиса Грегу.
- Исправлены и улучшены различные анимации с факелом (спасибо ElderGamer).
- Исправлена текстура внутренней стороны рта у некоторых персонажей (спасибо ElderGamer).
- Перед использованием предметов люди будут убирать оружие и магию.
- В мир не вставляются несуществующие бандиты BDT_10002_Addon_Bandit_M и BDT_10003_Addon_Bandit_H.
- Восстановлены визуальные эффекты заклинаний 'Огненный дождь', 'Волна смерти' и 'Армия тьмы'.
- Восстановлены визуальные эффекты заклинаний телепортации, лечения и превращения.
- Квест Лестера 'Травы для Константино' переименован в 'Сборщик трав', чтобы отличаться от квеста Константино.
- Исправлена невозможность закрыть квест 'Бизнес Фернандо' после ареста Фернандо.
- Реплики глашатая не будут прерываться, когда ГГ берет пиво с прилавка трактирщика.
- Исправлена невозможность получить пятую похлебку Теклы, если подраться с Буллко до вступления в гильдию.
- Исправлена недоступная реплика отшельника о невозможности дальнейшего обучения.
- В дневник добавляется запись об обучении у отшельника и Яна.
- Лестер расскажет о нападении людей-ящеров только с четвертой главы.
- Керолот правильно атакует охотников на драконов, если сказать ему об украденном кошельке ночью.
- Исправлена вероятность гибели Бладвина во время диалога перед битвой в пещере с краулерами.
- Исправлено возможное зацикливание анимации у людей, сидящих на стульях и скамейках.
- Исправлена область таверны в лагере бандитов, в которой были недоступны некоторые диалоги с Логаном.
- После принятия в маги у послушников нельзя спрашивать кто возглавляет монастырь.
- Исправлен тролль, застревавший в текстурах у раскопок магов Воды в четвертой главе.
- Квест 'Драконьи яйца' всегда открывается в дневнике для записей из диалогов с Беннетом и Ли.
- После открытия ворот замка стражник ворот перемещается со стены на землю и не погибает на глазах игрока.
- Исправлена невозможность давать лечебные зелья Марио на Ирдорате.
- Задействована неиспользуемая анимация выхода из состояния превращения (только с Union).
- ГГ сохраняет экипированное оружие при выходе из состояния превращения (только с Union, спасибо Slavemaster).
- Тела врагов не блокируют стрелы и магию во время анимации смерти (только с Union, спасибо Slavemaster).
- Исправлено зависание людей после получения урона в состоянии магического сна.
- Повторное применение заклинания сна продлевает время текущего магического сна.
- Молитвы Сержио и Гарвига не блокируют алтари Инноса для ГГ.
- Исправлено отсутствие опыта за убийства плывущих противников (только с Union, спасибо Slavemaster).
- Исправлен список ингредиентов для изготовления руны 'Буря' в дневнике.
- Исправлено отсутствие некоторых эффектов при высоких значениях FPS (только с Union, спасибо Slavemaster).
- Ищущие не будут использовать заклинания 'Огненный дождь' и 'Буря', если поблизости находятся другие Ищущие.
- Штрафы за преступления против Джека и Брайана не учитываются в городе после их ухода на маяк.
- Исправлена геометрия флагов на башнях города (спасибо N1kX).
- Исправлена текстура крыши гробницы Инубиса (спасибо N1kX).

<hr /><em>This discussion was created from the release <a href='https://github.com/dosinabox/g2nr_unofficial_update/releases/tag/v29'>v29 14.08.2023</a>.</em>

**Comment by Kor-Angar** [comment]
с паладином получилось стрёмно при игре за послушника

![20230819184840_1](https://github.com/dosinabox/g2nr_unofficial_update/assets/49514764/ca8ef472-95a5-4d9b-a543-2f75c4ec1236)

**Reply by Kor-Angar** [reply]
и он не стал почему то провожать героя мага в миненталь

**Reply by dosinabox** [reply]
Пришлось убрать его распорядок с ожиданием, из-за конфликтов с тренировками с Бабо. Сам диалог с началом сопровождения остался, но надо подходить к нему самому и сообщать о новых приказах.

**Comment by Kor-Angar** [comment]
Не Проще отменить тренировку Бабо? А при входе в новый мир ее снова восстановить. К тому же тренировку игрок все равно не застанет, так как это происходит ранее 7 утра.05:50, 20 августа 2023 г., AK ***@***.***>:
Пришлось убрать его распорядок с ожиданием, из-за конфликтов с тренировками с Бабо. Сам диалог с началом сопровождения остался, но надо подходить к нему самому и сообщать о новых приказах.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
-- Отправлено из мобильного приложения Яндекс Почты

**Reply by dosinabox** [reply]
Пока не получилось это всё согласовать. Теперь можно будет окинуть свежим взглядом.

**Comment by Kor-Angar** [comment]
паладины массово попадали со стены замка, так как теперь орки подходят под стену замка. там куда раньше им было запрещено подходить.

**Reply by dosinabox** [reply]
не помню чтобы в 29 менялось что-то на эту тему

**Reply by Kor-Angar** [reply]
наверное сильно раньше менялось.

**Reply by dosinabox** [reply]
раньше не жаловались

**Comment by Kor-Angar** [comment]
Если тебе нужны жалобы то надо заводить группу в вк каждый день будешь читать;-)В целом у меня это второй случай, с момента переноса.17:01, 21 августа 2023 г., AK ***@***.***>:
раньше не жаловались

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
-- Отправлено из мобильного приложения Яндекс Почты

**Reply by dosinabox** [reply]
В ВК меня уже давно нет. И какого переноса?

**Comment by Kor-Angar** [comment]
Переноса зоны до которой преследует орки.18:01, 21 августа 2023 г., AK ***@***.***>:
В ВК меня уже давно нет. И какого переноса?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
-- Отправлено из мобильного приложения Яндекс Почты

**Reply by dosinabox** [reply]
и когда этот перенос произошел? по-моему он был ещё до меня, в самых первых версиях Димуса

**Comment by Kor-Angar** [comment]
По моему это ты делала в прошлом году.Видимо что бы игроки не убивали орков с разбитого сооружения для штурма замка.18:14, 21 августа 2023 г., AK ***@***.***>:
и когда этот перенос произошел? по-моему он был ещё до меня, в самых первых версиях Димуса

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
-- Отправлено из мобильного приложения Яндекс Почты

**Reply by dosinabox** [reply]
Память тебя подводит. Это правка функции ZS_MM_Attack_Loop 2017 года, где изменили точку и расстояние до нее: https://github.com/dosinabox/g2nr_unofficial_update/commit/f8b8dd8f4fe5c170c1be1b26160eb0188a25490d#diff-328b12aacb249c78327e6225e1d76b9ad05bce53efd1e8230bdec8a78ada3be6L61

С тех пор эта проверка ни разу не менялась.

**Comment by Kor-Angar** [comment]
Значит память меня подводит :-)19:01, 21 августа 2023 г., AK ***@***.***>:
Память тебя подводит. Это правка 2017 года, где изменили точку и расстояние до нее: f8b8dd8#diff-328b12aacb249c78327e6225e1d76b9ad05bce53efd1e8230bdec8a78ada3be6L61
С тех пор эта проверка ни разу не менялась.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
-- Отправлено из мобильного приложения Яндекс Почты

**Comment by Kor-Angar** [comment]
Можно попробовать поменять настройки паладину который умирает в 4 главе, что бы он не падал, то есть в его инстанции прописать такой же аивер (?) как у Удара. 19:01, 21 августа 2023 г., AK ***@***.***>:
Память тебя подводит. Это правка 2017 года, где изменили точку и расстояние до нее: f8b8dd8#diff-328b12aacb249c78327e6225e1d76b9ad05bce53efd1e8230bdec8a78ada3be6L61
С тех пор эта проверка ни разу не менялась.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
-- Отправлено из мобильного приложения Яндекс Почты

**Reply by dosinabox** [reply]
Это не аивар, а fight_tactic FAI_NAILED. И у VLK_4145_Waffenknecht он уже прописан, как у Удара. У Сенграта тоже.

**Reply by dosinabox** [reply]
Привела им несколько орков по тарану, никто не падал.

**Reply by Kor-Angar** [reply]
![20230820041330_1](https://github.com/dosinabox/g2nr_unofficial_update/assets/49514764/61b64f9b-555a-4340-ba8c-8c04954df00b)

**Reply by Kor-Angar** [reply]
дисантники  без парашютов :-)

**Reply by Kor-Angar** [reply]
![20230820041524_1](https://github.com/dosinabox/g2nr_unofficial_update/assets/49514764/34fcf788-a378-448b-947d-76a19c1e499d)

**Comment by dosinabox** [comment]
> Ищущие не будут использовать заклинания 'Огненный дождь' и 'Буря', если поблизости находятся другие Ищущие.

Отменено.
